examples/Utils/b3Clock.cpp

```diff
@@ -112,16 +112,14 @@ void b3Clock::reset()
 unsigned long int b3Clock::getTimeMilliseconds()
 {
 #ifdef B3_USE_WINDOWS_TIMERS
-	LARGE_INTEGER currentTime;
-	QueryPerformanceCounter(&currentTime);
-	LONGLONG elapsedTime = currentTime.QuadPart - 
-		m_data->mStartTime.QuadPart;
-		// Compute the number of millisecond ticks elapsed.
-	unsigned long msecTicks = (unsigned long)(1000 * elapsedTime / 
-		m_data->mClockFrequency.QuadPart);
-	
-
-		return msecTicks;
+		LARGE_INTEGER currentTime, elapsedTime;
```

Comment:
I just made this code consistent with all other code segments using a very similar code segment here. But you are right, I should not have mixed it. I will undo it and maybe PR for it again so that this class is a bit cleaner. Ok?